tests/test_cgf.py

```diff
@@ -0,0 +1,71 @@
+import sys
+import numpy as np, pandas as pd, scipy.stats
+from vivarium import InteractiveContext, Artifact
+
+TIME_FIRST = 29
+TIME_SECOND =366-TIME_FIRST
+
+AGE_FIRST=(0.07671, 1.0)
+AGE_SECOND=(1.0, 5.0)
+
+ROUND_TO = 4
+
+def trans(inp, round_to):
+    return (inp - 10.0).round(round_to)
```

Comment:
Consider using `numpy.allclose` with `rtol=1e-4` as a cooler way to accomplish this https://numpy.org/doc/stable/reference/generated/numpy.allclose.html

Reply:
oops, i was reading too quickly; this is for pretty output, not for testing, right?  I get it now.  `f'{inp-10:.4f}'` will sometimes print prettier than rounding to 4 decimal places.